CHANGELOG.adoc

```diff
@@ -24,6 +24,7 @@
 - https://github.com/eclipse-syson/syson/issues/160[#160] [syson] Add explicit usage of node to call syside javascript file.
 - https://github.com/eclipse-syson/syson/issues/172[#172] [details] Add Membership#visibility attribute to Core properties in the Details view.
 - https://github.com/eclipse-syson/syson/issues/171[#171] [details] Add PortUsage#direction attribute to Core properties in the Details view.
+- https://github.com/eclipse-syson/syson/issues/177[#177] [diagrams] Missing reconnect tools for composition edges
```

Comment:
Could you the message for something like: `Add reconnect tools for composition edges` please?

Reply:
Done. The issue title and the commit message have been changed consequently.

Reply:
But could you also change the message in the changelog please ?

Reply:
No you didn't, in the changelog, the message is still `Missing reconnect tools for composition edges`

Reply:
I've just pushed my review changes now.

Reply:
I marked it as `Done` only to know that I agree and the change is going to be done and pushed with an amended commit.